peps/pep-0728.rst

```diff
@@ -0,0 +1,640 @@
+PEP: 728
+Title: TypedDict with Typed Extra Items
+Author: Zixuan James Li <p359101898@gmail.com>
+Sponsor: Jelle Zijlstra <jelle.zijlstra@gmail.com>
+Status: Draft
+Type: Standards Track
+Topic: Typing
+Content-Type: text/x-rst
+Created: 12-Sep-2023
+Python-Version: 3.13
+
+
+.. highlight:: rst
```

Comment:
Please remove this, the code blocks should default to Python instead.

Reply:
Was wondering about why that was happening. Thanks for spotting that!